<ui version="4.0" >
 <class>AddEventDialogClass</class>
 <widget class="QDialog" name="AddEventDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>AddEventDialog</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Event</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2" >
      <property name="fieldGrowthPolicy" >
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Event starts:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDateTimeEdit" name="editEventStart" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Event ends:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDateTimeEdit" name="editEventEnd" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QTextEdit" name="textEdit" />
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Recurrence:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QComboBox" name="comboRecurrence" >
        <item>
         <property name="text" >
          <string>not recurrent</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>every day</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>every weekday</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>every week</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>every month</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>every year</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Reminder</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4" >
      <property name="fieldGrowthPolicy" >
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="checkBox" >
        <property name="text" >
         <string>Enable reminder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Remind time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="comboRemind" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <item>
         <property name="text" >
          <string>on time</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>5 minutes before</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>10 minutes before</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>15 minutes before</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>30 minutes before</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>other</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Other:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDateTimeEdit" name="editRemindOther" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QCheckBox" name="checkBox_3" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Tray baloon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_2" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Sound</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Remind with:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOk" >
         <property name="text" >
          <string>&amp;OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCancel" >
         <property name="text" >
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
